packages/react-native-bridge/index.js

```diff
@@ -202,6 +202,19 @@ export function subscribeConnectionStatus( callback ) {
 	);
 }
 
+/**
+ * Subscribes a callback function to the 'onVoiceToContent' event.
+ * This event is triggered with markdown content that will be passed to the block editor
+ * to be converted into blocks.
+ *
+ * @param {Function} callback - The function to be called when the 'onVoiceToContent' event is triggered.
+ *                            This function receives markdown content as an argument.
+ * @return {Object} - The listener object that was added to the event.
+ */
+export function subscribeVoiceToContent( callback ) {
+	return gutenbergBridgeEvents.addListener( 'onVoiceToContent', callback );
+}
```

Comment:
Thank you for the feedback and the suggestions David! I agree, I'll work on changing the naming since it can be used for more things not related to "voice" input 👍